.github/workflows/test.yml

```diff
@@ -0,0 +1,18 @@
+on: [push, pull_request]
+name: Test
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [1.13.x, 1.16.x]
```

Comment:
All versions of go since 1.1?

Whilst I'm unsure of why people use older versions of Go, I want to ensure people know what this works against and when we stop supporting an older version I want that to be explicit.